packages/protocol/congestioncontrol/icca/scheduler/scheduler.go

```diff
@@ -78,9 +77,11 @@ func New(evictionState *eviction.State, isBlockAccepted func(models.BlockID) boo
 		blocks:                             memstorage.NewEpochStorage[models.BlockID, *Block](),
 		optsMaxBufferSize:                  300,
 		optsAcceptedBlockScheduleThreshold: 5 * time.Minute,
-		optsRate:                           5 * time.Millisecond,
+		optsRate:                           5 * time.Millisecond,      // measured in time per unit work
+		optsMaxDeficit:                     new(big.Rat).SetInt64(10), // must be >= max block work, but work is currently=1 for all blocks
```

Comment:
So an issuer can accumulate deficit to issue maximum 10 blocks, is this a reasonable default?

Reply:
I think it seems reasonable tbh.. If they have enough access mana this deficit will grow very fast anyway. I often used only one or two as max deficit in simulations for this value.